src/extension/background-script/actions/allowances/__tests__/delete.test.ts

```diff
@@ -68,6 +74,15 @@ describe("delete allowance", () => {
       .reverse()
       .sortBy("lastPaymentAt");
 
+    if (dbAllowance) {
+      paymentsCount = await db.payments
+        .where("host")
+        .equalsIgnoreCase(dbAllowance.host)
+        .count();
+
+      expect(paymentsCount).toEqual(0);
```

Comment:
Does it make sense to first check if payments exists i.e. 
```ts
expect(paymentsCount).toEqual(1);
```
and then check for `0`?

Reply:
Yeah, that's what I thought

Reply:
Cool, wanna add it?